### PR TITLE
Fix component governance packages

### PR DESCRIPTION
**Pull request by fearthecowboy**
Important details:
 - needed to use NPM's `overrides` capability, which is available with a newer version of node, so I updated to nodejs 16.x

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -25,14 +25,13 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
-      # Node 14 matches the version of Node used by VS Code when this was
-      # written, but it should be updated when VS Code updates its Node version.
+      # updated to Node 16
```
Comment (aleun) [inline .github/workflows/build.yml:28]: I'd prefer keeping the above comment, but just updating to say Node 16 so it's clear we're trying to sync with the VS Code Node version.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -171,7 +171,7 @@ steps:
       uploadRoslyn: false
       uploadTSLint: false
     # Don't open bugs for PR builds
-    condition: ne(variables['Build.Reason'], 'PullRequest')
+    # condition: ne(variables['Build.Reason'], 'PullRequest')
```
Comment (aleun) [inline azure-pipelines.yml:174]: This should be reverted before merging.

File: src/arduino/exampleProvider.ts [hunk 3]
```diff
@@ -43,7 +43,7 @@ export class ExampleProvider implements vscode.TreeDataProvider<ExampleItem> {
         this._exmaples = null;
         this._exampleManager.loadExamples().then((examples) => {
             this._exmaples = examples;
-            this._onDidChangeTreeData.fire();
+            this._onDidChangeTreeData.fire(null);
```
Comment (aleun) [inline src/arduino/exampleProvider.ts:46]: I'm guessing this is unrelated to anything CG related?

File: package.json [hunk 4]
```diff
@@ -660,6 +660,10 @@
     "uuid": "^3.0.1",
     "vscode-extension-telemetry": "0.1.6",
     "winreg": "^1.2.3",
-    "winston": "^2.3.1"
+    "winston": "^2.4.5"
+  },
+  "overrides": {
+    "async": "3.2.3",
```
Comment (aleun) [inline package.json:666]: Should this be `^` or `~` prefixed?

File: src/views/package.json [hunk 5]
```diff
@@ -36,5 +36,10 @@
     "ts-loader": "^4.5.0",
     "webpack": "^4.44.1"
   },
-  "dependencies": {}
-}
+  "dependencies": {},
+  "overrides": {
+    "qs": "6.10.3",
+    "async": "3.2.3",
```
Comment (aleun) [inline src/views/package.json:42]: Should this (and `qs` above) be `^` or `~` prefixed?

**Review by aleun — APPROVED** [review]

File: src/arduino/exampleProvider.ts (hunk 3 above)
Comment (fearthecowboy) [reply src/arduino/exampleProvider.ts:46]: This line broke when I updated the dependencies, so I patched it to continue working.

I'm not sure this code is even used, as it's called "Example Provider" - My assumption is that it's part of the original vscode extension template or something.

File: src/views/package.json (hunk 5 above)
Comment (fearthecowboy) [reply src/views/package.json:42]: Sure. I can do that

File: .github/workflows/build.yml [hunk 6]
```diff
@@ -25,14 +25,14 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
-      # Node 14 matches the version of Node used by VS Code when this was
-      # written, but it should be updated when VS Code updates its Node version.
+      # Node 16 matches the version of Node used by VS Code when this was
+-     # written, but it should be updated when VS Code updates its Node version.
```
Comment (BelKed) [inline .github/workflows/build.yml:29]: I guess the added `-` is just a typo, isn't it?

Comment (BelKed) [reply .github/workflows/build.yml]: After https://github.com/microsoft/vscode-arduino/pull/1493/commits/fb9c4d8336da895bc4f6df8552e3f5a307d446cd there is one space missing, so the comments aren't aligned...

Comment (benmcmorran) [reply .github/workflows/build.yml]: Fixed in https://github.com/microsoft/vscode-arduino/pull/1493/commits/fb9c4d8336da895bc4f6df8552e3f5a307d446cd

**Comment by benmcmorran** [comment]
Bypassing CLA requirement. The bot is broken, and Garrett works for Microsoft.
